README.md

```diff
@@ -420,6 +420,9 @@ git add RELEASE && git commit --amend
 
 Finally, push the branch and use it to create a PR. When the PR is merged, a GitHub release should be created with auto-generated release notes, as well as a git tag. Then the `publish-latest` CI job should trigger and if everything goes well, publish the new version of the repo's packages to `npm`.
```

Comment:
Could you please add a description under the chapter `Tests` how to run the ui tests?

Reply:
done

Reply:
oops - hold-on I just noticed something wrong. Will push an update soon

Reply:
Ok, should be good now. Since I was updating the tests chapter, I made some corrections/enhancements to the test coverage sub-section - I had observed that the provided command resulted in no coverage report.